packages/axios-extension/src/base/odata-request-error.ts

```diff
@@ -1,3 +1,15 @@
+import type { AxiosError } from 'axios';
+
+/**
+ * Casts an unknown error to an AxiosError.
```

Comment:
Isn't it more accurate to say that the this type guard narrows the type down to an AxiosError _if_ it is actually one? Not very pithy...

Reply:
https://github.com/SAP/open-ux-tools/pull/797/commits/73a5f5dd77aa97088071fae6b445d6786bc65561 better?